src/components/explorebox/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Box = styled.button`
```

Comment:
instead of a button here, you could make this styled compnent a div and wrap the whole thing in <a> tags for the link. Might look cleaner/goes better with semantic HTML